WordPress/gutenberg
# ColorPalette: Fix background image in RTL mode

## What?
Fix background image in the `ColorPalette` component when no color is picked in RTL mode.

## Why?
Better RTL experience 

## How?
Don't change the `background-position` value in the RTL css file.

## Testing Instructions
1. Fire up the storybook.
2. Go to `ColorPalette`
3. Change to RTL and deselect the color. The background should show white and gray squares.

## Screenshots or screencast 
| Before | After |
| - | - |
| ![color-palette-bug](https://user-images.githubusercontent.com/1415747/179546574-c197e4a8-c4b9-428b-977e-cb4b0af7f0e9.gif) | ![color-palette-fixed](https://user-images.githubusercontent.com/1415747/179546732-3fafa34b-aea4-4f72-a036-69384cf19909.gif) |
